cmd/wego/app/status/cmd.go

```diff
@@ -7,7 +7,7 @@ import (
 
 var Cmd = &cobra.Command{
 	Use:     "status <app-name>",
-	Short:   "Get status of an app",
+	Short:   "Get status of a workload under wego control",
```

Comment:
@bia are you OK with introducing the term `workload` here? It does match the `wego add` short description

Reply:
Thanks for the ping !
@davidstauffer and I need to sync on this.

Reply:
Can't resolve until we hear back from @bia and/or @davidstauffer

Reply:
Since this is in a help comment, I'm going to approve the PR from my perspective.  We can update the help text in the future if necessary. But I suggest trying to get ahold of Bianca and/or David but not blocking on that.

Reply:
I think I'd rather just change this to "Get status of an app under wego control" as it'll be in sync with other messages of this sort.